matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/message/PollType.kt

```diff
@@ -27,9 +27,15 @@ enum class PollType {
     @Json(name = "org.matrix.msc3381.poll.disclosed")
     DISCLOSED,
 
+    @Json(name = "m.poll.disclosed")
+    DISCLOSED_STABLE,
+
     /**
      * Results should be only revealed when the poll is ended.
      */
     @Json(name = "org.matrix.msc3381.poll.undisclosed")
-    UNDISCLOSED
+    UNDISCLOSED,
+
+    @Json(name = "m.poll.undisclosed")
+    UNDISCLOSED_STABLE
```

Comment:
I would rather rename the other to `UNDISCLOSED_UNSTABLE` and name this one `UNDISCLOSED`, since at the end we will have only this one.

Reply:
Done.